src/dosemu-downloaddos

```diff
@@ -234,11 +234,14 @@ if args.os:
     download_dos(dos_to_download, destination)
 
 if args.custom and args.destination:
-    url = args.custom
+    urls = args.custom
     destination = args.destination
     os.makedirs(destination, exist_ok=True)
     if not os.listdir(destination):
-        download_and_process(derive_url_list(url), destination, args.sha256sum if args.sha256sum else [])
+        if len(urls) == 1:
+            download_and_process(derive_url_list(urls[0]), destination, args.sha256sum if args.sha256sum else [])
```

Comment:
Is this still necessary?
I don't know what exactly it is
doing, but would the more uniform
approach of always "multiple urls"
case to work?

Reply:
There is some complex logic to derive the other URLs in case just one is supplied, this is possible because of very common filename patterns. I intend to split all the download and extraction logic from the FreeDOS logic that are both in this script. I'm not sure yet in which repository this download and extraction logic had best be.

Part of the download logic will be used by both the scripts to fetch FreeDOS packages and other third party/proprietary software.

Reply:
> There is some complex logic to derive the other URLs in case just one is supplied

Supplied by whom?
It is a conversion of 1 URL to
many. I just propose to move
that conversion somewhere
upper the layer, so that here
always multiple URLs come.